Libraries/LibWeb/HTML/Scripting/ModuleScript.cpp

```diff
@@ -133,18 +135,64 @@ WebIDL::ExceptionOr<GC::Ptr<ModuleScript>> ModuleScript::create_a_json_module_sc
     return script;
 }
 
+// https://html.spec.whatwg.org/multipage/webappapis.html#creating-a-webassembly-module-script
+// https://whatpr.org/html/9893/webappapis.html#creating-a-webassembly-module-script
+WebIDL::ExceptionOr<GC::Ptr<ModuleScript>> ModuleScript::create_a_webassembly_module_script(ByteString const& filename, ByteBuffer body_bytes, JS::Realm& realm, URL::URL base_url)
+{
+    // 1. If scripting is disabled for realm, then set bodyBytes to the byte sequence 0x00 0x61 0x73 0x6d 0x01 0x00 0x00 0x00.
+    // NOTE: This byte sequence corresponds to an empty WebAssembly module with only the magic bytes and version number provided.
+    if (HTML::is_scripting_disabled(realm))
+        ReadonlyBytes { { 0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00 } }.copy_to(body_bytes);
```

Comment:
This will not magically delete the rest of the body_bytes data, and the parser will most likely complain about it being invalid.
`body_bytes = body_bytes.slice(...)` probably?